.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
Should the about include a note about reaching out to support for help with your integration?

Reply:
The "about" section here is what gets shown in the sub-label for the issues drop down. I think adding the note about contacting support might be too much text for that view. 

![issue](https://user-images.githubusercontent.com/35243507/77776082-79e99300-701b-11ea-9c36-6faab3bc919f.png)